README.md

```diff
@@ -1,23 +1,58 @@
 [![CircleCI](https://circleci.com/gh/guardian/media-atom-maker.svg?style=svg)](https://circleci.com/gh/guardian/media-atom-maker)
 
-# media-atom-maker
-This app is a simple Play app for editing media atoms,
-and it's a publisher that pushes media atoms into a Kinesis stream.
+# Media-Atom-Maker
+This app a Play app for editing media atoms, and it's a
+publisher that pushes media atoms into a Kinesis stream.
 
 ## Setting up dev environment
-0. Set up a dev stack by generating a `media-atom-maker-dev.json` template from the corresponding `.yml` file
-in the cloudformation folder
-0. Run `./scripts/setup-dev-conf` to generate an `application.conf` file (`cd` into `scripts` first)
-0. You will need read access to S3 files in the `workflow` profile.
-0. The project makes use of [panda auth](https://github.com/guardian/pan-domain-authentication),
-so follow the instructions [here](https://github.com/guardian/dev-nginx#nginx-dev-setup) to set up nginx locally.
+0. Run `./scripts/setup-dev-conf` to generate an `application.conf` file.
+0. You need to pass to this script the name of the stack that you want to use
```

Comment:
the second arg is the aws profile for panda access, we default the profile for running the app to [media-service](https://github.com/guardian/media-atom-maker/blob/master/scripts/setup-dev-conf.sh#L15)